notebooks/304-pytorch-post-training-quantization/304-pytorch-post-training-quantization.ipynb

```diff
@@ -0,0 +1,698 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "afa7e952",
+   "metadata": {},
+   "source": [
+    "# Post-Training Quantization of PyTorch models with NNCF\n",
+    "\n",
+    "This notebook is based on 'ImageNet training in PyTorch' example.\n",
```

Comment:
I don't like it and I propose to remove it or add the following text at the end of the notebook: "Refer to 'ImageNet training in PyTorch' [example] for 8-bit quantization with the fine-tuning by NNCF."